src/pages/page-names.constants.ts

```diff
@@ -14,6 +14,10 @@ export const CAT_A_MOD1 = {
   REKEY_UPLOAD_OUTCOME_PAGE: 'RekeyUploadOutcomePage',
 };
 
+export const CAT_A_MOD2 = {
+  REKEY_UPLOAD_OUTCOME_PAGE: 'RekeyUploadOutcomePage',
```

Comment:
Should this be `RekeyReasonCatAMod2Page `

Reply:
No, I don't believe so. This page does not get refactored for each category.